gems/pending/spec/appliance_console/logfile_configuration_spec.rb

```diff
@@ -1,51 +1,119 @@
+      expect(subject).to receive(:agree).with(/Confirm continue with these upda.*/).and_return(true)
       expect(subject.ask_questions).to be true
     end
 
-    it "returns false when user does not confirm a new disk" do
-      expect(subject).to receive(:agree).and_return(false)
-      expect(subject).to_not receive(:ask_for_disk)
+    it "returns false when the user does not confirm the updates" do
+      expect(subject).to receive(:ask_for_integer).and_return(99)
+      expect(subject).to receive(:ask_for_disk).and_return(double(@spec_name, :size => "9999999", :path => "fake disk"))
+      expect(subject).to receive(:agree).with(/Change the saved logrotate count.*/).and_return(true)
+      expect(subject).to receive(:agree).with(/Configure a new logfile disk vol.*/).and_return(true)
+      expect(subject).to receive(:agree).with(/Confirm continue with these upda.*/).and_return(false)
       expect(subject.ask_questions).to be false
     end
 
-    it "returns false when user does not confirm the selection" do
-      expect(subject).to receive(:agree).with("Configure a new logfile disk volume? (Y/N):").and_return(true)
-      expect(subject).to receive(:agree).with(/Continue with disk:/).and_return(false)
-      expect(subject).to receive(:ask_for_disk).and_return(double(:@spec_name, :size => "22", :path => "fake disk"))
-
+    it "returns false when the user did not request a new disk or a new logrotate count" do
+      expect(subject).to_not receive(:ask_for_integer)
+      expect(subject).to_not receive(:ask_for_disk)
+      expect(subject).to receive(:agree).with(/Change the saved logrotate count.*/).and_return(false)
+      expect(subject).to receive(:agree).with(/Configure a new logfile disk vol.*/).and_return(false)
+      expect(subject).to_not receive(:agree).with(/Confirm continue with these upda.*/)
       expect(subject.ask_questions).to be false
     end
   end
 
   describe "#activate" do
+    let(:expected_miq_logs_conf) do
+      <<-EOT.strip_heredoc
+        /var/www/miq/vmdb/log/*.log /var/www/miq/vmdb/log/apache/*.log {
+          daily
+          dateext
+          missingok
+          rotate 3
+          notifempty
+          compress
+          copytruncate
+          prerotate
+            source /etc/default/evm; /bin/sh ${APPLIANCE_SOURCE_DIRECTORY}/logrotate_free_space_check.sh $1
+          endscript
+          lastaction
+            /sbin/service httpd reload > /dev/null 2>&1 || true
+          endscript
+        }
+      EOT
+    end
+
     it "stops and starts evm and configures the logfile disk" do
-      expect(ApplianceConsole::LogicalVolumeManagement).to receive(:new).and_return(double(@spec_name, :setup => true))
+      subject.logrotate_count = 3
+      subject.requested_new_disk = true
+      subject.requested_new_logrotate_count = true
+
+      expect(ApplianceConsole::LogicalVolumeManagement)
+        .to receive(:new).and_return(double(@spec_name, :setup => true))
```

Comment:
@carbonin Yes rubocop flagged some of them as 

> Line is too long.

So I thought it read better to have all the expects formatted the same. I can change it so only those that are too long will be broken across multiple lines.